WalletWasabi.Fluent/Views/Wallets/Home/History/HistoryTable.axaml

```diff
@@ -113,10 +113,56 @@
     </DataBox.Styles>
 
     <DataBox.Columns>
+      <!-- Indicators -->
+      <DataBoxTemplateColumn Width="Auto"
+                             MinWidth="120"
+                             CanUserSort="False">
```

Comment:
IMO there should be a column name, it looks a bit weird to me without it.

![image](https://user-images.githubusercontent.com/16364053/142394729-a73fd597-6a2f-4cbc-b5c8-1753612128f6.png)